.github/workflows/testing-integration.yml

```diff
@@ -21,24 +21,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [ '7.4', '7.3' ]
-        wordpress: [ '5.9', '5.8', '5.7.2', '5.6.2' ]
+        php: [ '8.0', '7.4', '7.3' ]
+        wordpress: [ '6.1', '6.0', '5.9', '5.8', '5.7', '5.6' ]
```

Comment:
@justlevine we have to be careful with this Matrix because WordPress doesn't support all versions of WP with all versions of PHP in Docker Hub. 

So we can't have the combo `php8.0` with `WP 5.6` for example. 

We _might_ want to just get rid of the matrix and be more explicit with the list of includes, because this has caused me pain in the past as well 🤷🏻‍♂️

Reply:
The matrix feels more convenient, but the reality is Docker hub just doesn't have images for all the matrix versions

Reply:
I caught that in https://github.com/wp-graphql/wp-graphql/pull/2605/commits/49bfd20810a56cdb7c846e026a0b64925a503c2a .

The current run is failing just because it seems somethings are breaking in 6.1 (new caching features I bet)

Reply:
uh oh!

Reply:
running locally in 6.1 to investigate

Reply:
Hmm...seems to be related to "restricted posts" 🤔 

```bash
There were 3 failures:

---------
1) PostObjectQueriesTest: Restricted posts | #0
 Test  tests/wpunit/PostObjectQueriesTest.php:testRestrictedPosts
Failed asserting that two arrays are equal.
- Expected | + Actual
@@ @@
'id' => 'cG9zdDoxNzQ4'
'title' => 'RestrictedPost from author: 0'
'status' => 'future'
-        'author' => null
+        'author' => Array (...)
'content' => '<p>Test Content</p>\n'
)
)
#1  /var/www/html/wp-content/plugins/wp-graphql/tests/wpunit/PostObjectQueriesTest.php:1632
#2  /var/www/html/wp-content/plugins/wp-graphql/vendor/bin/codecept:120

---------
2) PostObjectQueriesTest: Restricted posts | #3
 Test  tests/wpunit/PostObjectQueriesTest.php:testRestrictedPosts
Failed asserting that two arrays are equal.
- Expected | + Actual
@@ @@
'id' => 'cG9zdDoxNzUx'
'title' => 'RestrictedPost from author: 0'
'status' => 'draft'
-        'author' => null
+        'author' => Array (...)
'content' => '<p>Test Content</p>\n'
)
)
#1  /var/www/html/wp-content/plugins/wp-graphql/tests/wpunit/PostObjectQueriesTest.php:1632
#2  /var/www/html/wp-content/plugins/wp-graphql/vendor/bin/codecept:120

---------
3) PostObjectQueriesTest: Restricted posts | #6
 Test  tests/wpunit/PostObjectQueriesTest.php:testRestrictedPosts
Failed asserting that two arrays are equal.
- Expected | + Actual
@@ @@
'id' => 'cG9zdDoxNzU0'
'title' => 'RestrictedPost from author: 0'
'status' => 'pending'
-        'author' => null
+        'author' => Array (...)
'content' => '<p>Test Content</p>\n'
)
)
#1  /var/www/html/wp-content/plugins/wp-graphql/tests/wpunit/PostObjectQueriesTest.php:1632
#2  /var/www/html/wp-content/plugins/wp-graphql/vendor/bin/codecept:120

ERRORS!
Tests: 641, Assertions: 4669, Errors: 2, Failures: 3.

```